linux/power_operation_script/set_power_script.yml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check soft power operation
+# # Parameters:
+#   power_ops: power script operation
+#   power_script_type: default or custom
+
+- name: "Start test {{ power_script_type }} {{ power_ops }}"
+  ansible.builtin.debug:
+    msg:
+      - "Start test {{ power_script_type }} {{ power_ops }}"
+
+- name: "Set {{ power_script_type }} script for testcase {{ power_script_ops }}"
+  include: set_power_script_{{ power_script_type }}.yml
+  vars:
+    power_ops: "{{ power_ops }}"
+
+- name: "Check the current configuration for testcase {{ power_script_ops }}"
+  ansible.builtin.command: "{{ vmware_toolbox_cmd_path }} script {{ power_ops }} current"
+  register: power_script_result
+  delegate_to: "{{ vm_guest_ip }}"
+
+- name: "Print the current configuration for testcase {{ power_script_ops }}"
+  ansible.builtin.debug: var=power_script_result.stdout
+
+- name: "Check current configuration is expected"
+  ansible.builtin.assert:
+    that:
+      - power_script_result.stdout.strip() == script_file_path
+    fail_msg: "The configuration for {{ power_script_ops }} is {{ power_script_result.stdout }}, not as expected {{ script_file_path }}."
+    success_msg: "The configuration for {{ power_script_ops }} is {{ power_script_result.stdout }}, which is as expected."
```

Comment:
Change `The configuration for {{ power_script_ops }}` to `The {{ power_script_ops }} script`.
And change `{{ power_script_result.stdout }}` to `{{ power_script_result.stdout | default('') }}`